.github/workflows/java.yml

```diff
@@ -23,6 +23,9 @@ jobs:
             scala: 2.11.8
           - spark: 3.0.3
             scala: 2.11.8
+        include:
```

Comment:
You need to enable Scala 2.13 in this yml when compile Sedona so the Scala/Java test can actually run. There are some examples in the yml file.

Reply:
Doesn't 
```
run: if [ ${SPARK_VERSION:0:1} == "3" ]; then mvn -q clean install -Dscala=${SCALA_VERSION:0:4} -Dspark=3.0 ; else mvn -q clean install -Dscala=${SCALA_VERSION:0:4} -Dspark=2.4 ; fi
```
already handle that? It looks like that test did run with scala 2.13 in the actions (I see deprecation warnings). Also, since the patch version is ignored in the tests should the scala version just be set to the minor version in the matrix?

Reply:
Sorry, my bad. Forgot this.